src/ioctls/vm.rs

```diff
@@ -610,6 +610,49 @@ impl VmFd {
         Ok(())
     }
 
+    /// Enable the specified capability as per the `KVM_ENABLE_CAP` ioctl.
+    ///
+    /// See the documentation for `KVM_ENABLE_CAP`.
+    ///
+    /// This ioctl returns Ok() when the capability is successfully enabled,
```

Comment:
I don't think you need the comment about the return type so we can keep the same comment style on all methods. Methods in this crate only specify the return type when it's not Result<()>.